interface/views.py

```diff
@@ -74,6 +74,24 @@ def homepage(request):
                    'logout_url': redirect(logout_view).url})
 
 
+@login_required
+def review(request, pk):
+    if request.POST and request.user.is_staff:
+        marks = re.findall('^(\+|-[0-9]+\.[0-9]+):',  # noqa: W605
```

Comment:
Can we not add `noqa` if we can avoid it? Those warnings are there for a reason.

Also, try this indent, I think it's nicer:

```python
marks = re.findall(
    '^(\+|-[0-9]+\.[0-9]+):',
    request.POST['review-code'],
    re.MULTILINE,
)
```

Also, Python has `r''` ([raw strings](https://docs.python.org/3/library/re.html#raw-string-notation)) for regular expressions, which treat escapes differently. In other words, your backslash escapes don't do what you expect them to do.